src/ui/qgsvectorlayerpropertiesbase.ui

```diff
@@ -176,7 +176,7 @@
            <string>Source Fields</string>
```

Comment:
It might be good to modify the title too here, as QGIS currently lies! :wink: This panel isn't only listing source fields, in fact it'll list joined fields, as well as expression-based virtual fields too. Let's remove source and just label it 'Fields'